modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/utils/ServerUtilities.java

```diff
@@ -126,14 +130,20 @@ public static HttpRequestBase buildHeaders(HttpRequestBase httpRequestBase,
 	}
 
 	public static Map<String, String> postData(EndPointInfo endPointInfo,
-	                                           Map<String, String> headers)
+											   Map<String, String> headers)
 			throws IDPTokenManagerException {
 
 		HTTP_METHODS httpMethod = endPointInfo.getHttpMethod();
 		String url = endPointInfo.getEndPoint();
-		JSONObject params = endPointInfo.getRequestParams();
+
+		String params = "";
```

Comment:
Why is variable "params" assigned an empty string? If there's no specific reason to do this assignment, we should probably keep the value null assigned to the aforementioned variable as it's default value.